packages/builder/src/steps/import.ts

```diff
@@ -62,6 +72,16 @@ export default {
     const importLabel = packageState.currentLabel?.split('.')[1] || '';
     debug('exec', config);
 
+    /*
+     * Source preset reference
+     * Checks for preset reference in source name, anything after the '@' references a preset.
+     * If a preset reference is found in the package name by default it will take preference over a defined preset.
+     */
+    if (config.source.includes('@')) {
```

Comment:
might make sense to turn the package name into a proper data structure at this point, rather than having this splitting/math happening all over the place